convert2rhel/actions/backup_system.py

```diff
@@ -0,0 +1,38 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, redhatrelease, repo
+
+
+loggerinst = logging.getLogger(__name__)
+
+
+class BackupSystem(actions.Action):
+    id = "BACKUP_SYSTEM"
+
+    def run(self):
+        """Backup system release file before starting conversion process"""
+        loggerinst.task("Prepare: Backup System")
+
+        super(BackupSystem, self).run()
+
+        redhatrelease.system_release_file.backup()
+        redhatrelease.os_release_file.backup()
```

Comment:
Those two needs to be inside a `try-except` block, as it can raise `logger.critical`.

Reply:
I'll add these in